test/controllers/merchants_controller_test.rb

```diff
@@ -0,0 +1,97 @@
+require "test_helper"
+require "pry"
+
+describe MerchantsController do
+  
+  #let(:merchant) { merchants(:brad) }
+
+  describe "index" do
+    it "responds with success when there are many merchants" do
+      get merchants_path
+      must_respond_with :success
+    end
+  
+  
+    it "responds with success when there are no merchants" do
```

Comment:
If there are merchants in your YAML, you'll need to delete all of them first within your "it" block before the rest of the logic